mfp.py

```diff
@@ -1,28 +1,30 @@
-print('Hi dude!')
+print('\nHi dude!')
+print('Whats your name?\n')
```

Comment:
Actually you can rework this piece using multi-line string as:
```python
hello_message = '''
Hi dude!
What's your name?
'''
print(hello_message)
```
This should give you the same result, but you don't have to care about line breaks `\n`